### PR TITLE
doc-kit run (fetch updates & replace entities) 

### PR DESCRIPTION
**Pull request by taroth21**
### Description

Applying the latest changes from doc-kit (metadata updates for assembly files + ALP-related entity updates). 

### Are there any relevant issues/feature requests?

* jsc#DOCTEAM-1332

### CODE REVIEW

File: articles/SLE_Micro_admin.asm.xml [hunk 1]
```diff
@@ -200,7 +200,7 @@
         <phrase>&power;</phrase>
       </meta>
       <meta name="productname" its:translate="no">
-        <!-- enter product name and version --><productname version="5.4">&slema;</productname>
+        <!-- enter product name and version --><productname version="5.4">&slm;</productname>
```
Comment (lvicoun) [inline articles/SLE_Micro_admin.asm.xml:203]: Here &slema; should stay as it is a 5.4 doc that is still named SLE Micro.

Comment (taroth21) [reply articles/SLE_Micro_admin.asm.xml:203]: Good catch, thanks!

File: common/product-entities.ent [hunk 2]
```diff
@@ -4,9 +4,9 @@
 <!-- PRODUCT NAME AND VERSIONS -->
 
 <!-- Use "&generic;" for everything not ALP and SLE Micro -->
-<!ENTITY productname            "<phrase xmlns='http://docbook.org/ns/docbook'><phrase os='alp-dolomite'>&suse; &productnameshort;</phrase><phrase os='slemicro'>&slem;</phrase><phrase os='generic'>&suselinux;</phrase><phrase os='sles'>&sles;</phrase></phrase>">
-<!ENTITY productnameshort       "<phrase xmlns='http://docbook.org/ns/docbook'><phrase os='alp-dolomite'>&alp-dolomite;</phrase><phrase os='slemicro'>&slema;</phrase><phrase os='generic'>&suselinux;</phrase><phrase os='sles'>&slea;</phrase></phrase>">
-<!ENTITY productnumber          "<phrase xmlns='http://docbook.org/ns/docbook'><phrase os='alp-dolomite'>0.1</phrase><phrase os='slemicro_5.4'>5.4</phrase><phrase os='slemicro_5.5'>5.5</phrase><phrase os='slemicro_6.0'>6.0</phrase><phrase os='sles'>15.5</phrase></phrase>">
+<!ENTITY productname            "<phrase xmlns='http://docbook.org/ns/docbook'><phrase os='slemicro'>&slm;</phrase><phrase os='generic'>&suselinux;</phrase><phrase os='sles'>&sles;</phrase></phrase>">
```
Comment (lvicoun) [inline common/product-entities.ent:7]: I think we need do add another <phrase> that will have &slm; as in this case all docs (even the 5.4 adn 5.5) will have the new product name :(

File: concepts/rootless-podman-limitations.xml [hunk 3]
```diff
@@ -22,7 +22,7 @@
   <itemizedlist>
     <listitem>
       <para>
-        &slem; does not allow regular users, thus making it impossible to run
+        &slm; does not allow regular users, thus making it impossible to run
```
Comment (lvicoun) [inline concepts/rootless-podman-limitations.xml:25]: I believe this should also be &productname; and the corresponding listitem should be profiled for Micro.

File: concepts/snapper-basic-concept.xml [hunk 4]
```diff
@@ -156,9 +156,9 @@
   </varlistentry>
   </variablelist>
   <important>
-    <title>Installation and administration snapshot types do not apply to &slem;</title>
+    <title>Installation and administration snapshot types do not apply to &slm;</title>
     <para>Installation and administration snapshot types do not apply to transactional 
-      systems, such as &slem;. For more information on &slema; snapshots, refer to
+      systems, such as &slm;. For more information on &slm; snapshots, refer to
```
Comment (lvicoun) [inline concepts/snapper-basic-concept.xml:161]: Similar situation here. I'll reach out to Daria and Dima regarding this.

File: tasks/enable-podman.xml [hunk 5]
```diff
@@ -92,7 +92,7 @@
     <note>
       <title>Limitations of rootless containers</title>
       <para>
-        Running a container with &podman; in rootless mode on &slema; may fail,
+        Running a container with &podman; in rootless mode on &slm; may fail,
```
Comment (lvicoun) [inline tasks/enable-podman.xml:95]: Again. If this article is related to Micro, then we should use productname here and hide the note for other products.

File: tasks/micro-cloud-useradd.xml [hunk 6]
```diff
@@ -17,7 +17,7 @@
   </info>
   <para>
     You can use the <command>useradd</command> command to add users to an
-    already running system. However, as &slema; is Btrfs-based, the
+    already running system. However, as &slm; is Btrfs-based, the
```
Comment (lvicoun) [inline tasks/micro-cloud-useradd.xml:20]:
```suggestion
    already running system. However, as &productname; is Btrfs-based, the
```

**Review by lvicoun — CHANGES_REQUESTED** [review]

File: articles/SLE-Micro-public-cloud.asm.xml [hunk 7]
```diff
@@ -67,7 +67,7 @@
       </meta>
       <!-- enter product name and version -->
       <meta name="productname" its:translate="no">
-        <productname version="slemicro">&slem;</productname>
+        <productname version="slemicro">&slm;</productname>
```
Comment (tbazant) [inline articles/SLE-Micro-public-cloud.asm.xml:70]: is this a correct version?

File: articles/cockpit-alp.asm.xml [hunk 8]
```diff
@@ -172,7 +172,7 @@ type="application/relax-ng-compact-syntax"?>
       </meta>
       <!-- enter product name and version -->
       <meta name="productname">
-        <productname version="slemicro">&slem;</productname>
+        <productname version="slemicro">&slm;</productname>
```
Comment (tbazant) [inline articles/cockpit-alp.asm.xml:175]: version?

File: common/product-entities.ent [hunk 9]
```diff
@@ -4,9 +4,9 @@
...
+<!ENTITY productnameshort       "<phrase xmlns='http://docbook.org/ns/docbook'><phrase os='slemicro'>&slm;</phrase><phrase os='generic'>&suselinux;</phrase><phrase os='sles'>&slea;</phrase></phrase>">
+<!ENTITY productnumber          "<phrase xmlns='http://docbook.org/ns/docbook'><phrase os='slemicro_5.4'>5.4</phrase><phrase os='slemicro_5.5'>5.5</phrase><phrase os='slemicro_6.0'>6.0</phrase><phrase os='sles'>15.5</phrase></phrase>">
```
Comment (tbazant) [inline common/product-entities.ent:9]: profiling for alp-dolomite is lost, shall we replace it with &slcs; or rather cancel Dolomite documentation?

**Review by tbazant — APPROVED** [review]
small nits, otherwise great job!

File: common/product-entities.ent (hunk 9 above)
Comment (taroth21) [reply common/product-entities.ent:9]: Yes, the product name is gone now. We need to replace the Dolomite docs with a very basic (product) description of SLCS as quickly as possible (see the second part of the description in DOCTEAM-1332). Half a page is good enough for the beginning.

File: common/product-entities.ent (hunk 2 above)
Comment (taroth21) [reply common/product-entities.ent:7]: At least, I thought it was the plan to copy over the articles for Micro 5.5 and older to doc-sle? But looking at the docserv2 config, I see that some of the content fro 5.5 is still built from doc-modular (and is included via `<internal>`  in that config).

That makes it difficult now. I think we need to discuss that next Tue.

File: articles/SLE-Micro-public-cloud.asm.xml (hunk 7 above)
Comment (taroth21) [reply articles/SLE-Micro-public-cloud.asm.xml:70]: @janajaeger ?

File: articles/SLE_Micro_admin.asm.xml (hunk 1 above)
Comment (taroth21) [reply articles/SLE_Micro_admin.asm.xml:203]: Depends on the answer to my comment below (if we keep building Micro 5.5. and 5.4 docs from both `doc-sle` and `doc-modular`)

File: common/product-entities.ent (hunk 2 above)
Comment (lvicoun) [reply common/product-entities.ent:7]: Yes, some docs were already moved do doc-modular as it happened before that decision was made.

File: articles/cpu-check-temperature.asm.xml [hunk 10]
```diff
@@ -51,7 +51,7 @@
         <phrase>&aarch64;</phrase>
       </meta>
       <meta name="productname">
-        <productname version="1">&alp;</productname>
+        <productname version="1">&slfo;</productname>
```
Comment (tbazant) [inline articles/cpu-check-temperature.asm.xml:54]:
```suggestion
```

File: articles/subnet-manager-configuring.asm.xml [hunk 11]
```diff
@@ -57,7 +57,7 @@
       </meta>
       <meta name="productname" its:translate="no">
         <productname version="15-SP5">&sles;</productname>
-        <productname version="1.0">&alpshort;</productname>
+        <productname version="1.0">&slfo;</productname>
```
Comment (tbazant) [inline articles/subnet-manager-configuring.asm.xml:60]:
```suggestion
```

File: common/generic-entities.ent [hunk 12]
```diff
@@ -82,11 +82,11 @@
 <!ENTITY bci-product            "&slea; Base Container Images">
 
 
-<!-- ALP -->
-<!ENTITY alp                    "Adaptable Linux Platform">
-<!ENTITY alpshort               "ALP">
-<!ENTITY alp-dolomite           "&alpshort;&nbsp;Dolomite">
-
+<!-- ALP/SUSE Linux Framework One -->
+<!ENTITY slfo                   "&suselinux; Framework One">
```
Comment (tbazant) [inline common/generic-entities.ent:86]:
```suggestion
```

**Review by tbazant — CHANGES_REQUESTED** [review]
added more suggestions - removing &alp*

File: articles/SLE-Micro-public-cloud.asm.xml (hunk 7 above)
Comment (taroth21) [reply articles/SLE-Micro-public-cloud.asm.xml]: Please see https://github.com/SUSE/doc-modular/pull/319/commits/321636c71fef1c7b4cba660596bd2ad7b0e6de2a

File: concepts/rootless-podman-limitations.xml (hunk 3 above)
Comment (taroth21) [reply concepts/rootless-podman-limitations.xml]: https://github.com/SUSE/doc-modular/pull/319/commits/93805b9febd72a0068ed440b5fd4f2ad175457d5

File: tasks/enable-podman.xml (hunk 5 above)
Comment (taroth21) [reply tasks/enable-podman.xml:95]: https://github.com/SUSE/doc-modular/pull/319/commits/93805b9febd72a0068ed440b5fd4f2ad175457d5

File: concepts/snapper-basic-concept.xml (hunk 4 above)
Comment (taroth21) [reply concepts/snapper-basic-concept.xml]: https://github.com/SUSE/doc-modular/pull/319/commits/93805b9febd72a0068ed440b5fd4f2ad175457d5

File: articles/systemd-timer-working-with.asm.xml [hunk 13]
```diff
@@ -58,13 +58,10 @@
...
-        <productname version="ALP">&sles;</productname>
-        <productname version="15-SP5">&sles;</productname>
-        <productname version="15-SP4">&sles;</productname>
-        <productname version="15-SP3">&sles;</productname>
-        <productname version="15-SP2">&sles;</productname>
-        <productname version="15-SP1">&sles;</productname>
-        <productname version="15-GA">&sles;</productname>
+        <productname version="5.4">&productname;</productname>
+        <productname version="5.5">&productname;</productname>
+        <productname version="6.0">&productname;</productname>
```
Comment (tbazant) [inline articles/systemd-timer-working-with.asm.xml:63]: Is `os="slemicro"` missing here?

**Review by tbazant — CHANGES_REQUESTED** [review]
LG but one glitch...

File: articles/systemd-timer-working-with.asm.xml (hunk 13 above)
Comment (lvicoun) [reply articles/systemd-timer-working-with.asm.xml]: Fixed.

**Pull request merged**